docs/build/get-started.md

```diff
@@ -286,7 +289,13 @@ For more information about all things Aqua, see the [Aqua book](/docs/aqua-book/
 
 **Scaffolding Options**
 
-Instead of the *minimal* scaffold chosen at the outset of this section, we can opt for an extended project setup for either Typescript or Javascript. Before we go exploring, a quick review of how Fluence and Aqua work might be in order: All communication with distributed services is over libp2p. Hence, you need a (p2p) client peer, rather than an HTTP client, to interact with the peers hosting your service(s). Choosing the *minimal* scaffolding setup provides you with a setup suitable to utilize a one-shot client-peer builtin to Fluence CLI. The TS/JS, setup, on the other hand, provides you with the scaffolding to create a client peer with [Fluence js-client](https://github.com/fluencelabs/js-client) that can run in the browser or as a node app. See Table 1.
+Instead of the *minimal* scaffold chosen at the outset of this section, we can opt for an extended project setup for either Typescript or Javascript.
+
+Before we go exploring, a quick review of how Fluence and Aqua work might be in order: All communication with distributed services is over libp2p. Hence, you need a (p2p) client peer, rather than an HTTP client, to interact with the peers hosting your service(s).
+
+Choosing the *minimal* scaffolding setup provides you with a setup suitable to start from scratch, write some functions and call them via Fluence CLI. In that sense, that scenario is a "one-shot" one, as opposed to long-running backend or frontend applications.
```

Comment:
```suggestion
Choosing the *minimal* scaffolding setup provides you with a setup suitable to start from scratch, write some functions and call them via Fluence CLI which provides a simple, "one-shot" client peer experience.
```

Reply:
I'm not sure what is "client peer", or "one-shot experience". Sounds a bit weird to me 🤔

Reply:
doesn't read well, i agree. just drop the one-shot reference?

Reply:
Well, it's in the table above. I think i'll remove it everywhere, yes.